.github/workflows/linux.yml

```diff
@@ -271,24 +295,29 @@ jobs:
     defaults:
       run:
         shell: bash
-    runs-on: ubuntu-20.04
+    runs-on: aks-linux-4-cores
```

Comment:
looks like it's enough to use default runner for this job, since it's fast enough

Reply:
Oops, okay, but let's leave it as it is here right now, it is cheaper anyway (